app/styles.css

```
@@ -7,9 +7,9 @@
 
 @theme {
   --font-*: initial;
-  --font-mono: Inconsolata, Menlo, Chicago, Geneva;
+  --font-mono: Inconsolata, Menlo, Czikago, Geneva;
```

Stylelint (17.13.0)

[error] Expected "Inconsolata" to be "inconsolata" (value-keyword-case)

(value-keyword-case)